### ISSUE TITLE
Vaulted backup service owner blob storage

### ISSUE THREAD

**Issue by bengtfredh** [Altinn/altinn-platform#869]
We need to add backupvault to service owner extended functionality storage account module.

**Comment by tjololo** [comment]
As far as I can see this is done @bengtfredh ?

**Comment by bengtfredh** [comment]
> As far as I can see this is done [@bengtfredh](https://github.com/bengtfredh) ?

No, it was referencing this https://dev.azure.com/brreg/altinn-apps-ops/_git/terraform-modules?path=/modules/storageaccount

**Comment by tjololo** [comment]
ah sorry, I see

**Comment by ootneim** [comment]
Duplicate: https://github.com/Altinn/altinn-platform/issues/1253
